puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/packaging_build_rpm.groovy

```diff
@@ -0,0 +1,77 @@
+pipeline {
+    agent { label 'rpmbuild' }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage("Setup Environment") {
+            steps {
+                git(url: 'https://github.com/theforeman/foreman-packaging/', branch: env.branch)
+                setup_obal()
+            }
+        }
+
+        stage("Apply Pull Request Changes") {
+            when {
+                expression { env.pr_git_url != null && env.pr_git_ref != null }
+            }
+            steps {
+                sh("""
+                    git remote add pr ${env.pr_git_url}
+                    git fetch pr
+                    git merge pr/${env.pr_git_ref}
+                """)
+            }
+        }
+
+        stage("Build RPM") {
+            when {
+                anyOf {
+                    expression { env.pr_git_url != null && env.scratch == 'true' }
+                    expression { env.pr_git_url == null }
+                }
+            }
+            steps {
+                script {
+                    sh("mkdir -p ${env.WORKSPACE}/rel-eng/build")
+
+                    def build_package_tito_releaser_args = "-o ${env.WORKSPACE}/rel-eng/build/"
+                    def build_package_scratch = false
```

Comment:
Nothing calls this job for scratch builds. Again a leftover from the PR testing.